src/pages/index.page.jsx

```diff
@@ -6,56 +6,69 @@ export default function Home() {
   return (
     <Layout pageTitle="Home">
       <div className="home">
-        <div className="home-top">
-          <div className="home-top__text-column">
-            <h1>Accessibility Cheatsheets for Components</h1>
-            <p className="home-top__top-description">
-              Intro copy goes here, such as: Build accessible components more
-              accurately and in less time with these easy-to-read cheatsheets
-              based on WCAG and ARIA best practices. Check back often as
-              we&#39;re adding new components frequently!
-            </p>
-          </div>
-          <div>
-            <img
-              className="home-top__top-image"
-              src="/home-header-image.svg"
-              alt="Illustration of dog surrounded by accessible components"
-            />
+        <div className="home-wrapper">
+          <div className="home-top">
+            <div className="home-top__text-column">
+              <h1>Accessibility Cheatsheets for Components</h1>
+              <p className="home-top__top-description">
```

Comment:
This text has a line height of 28px in Figma.

Reply:
Thank you again! Fixed here: https://github.com/sparkbox/accessible-components/pull/58/commits/c3cfeb7886d0f0f9e6207a22ad40811f717e2d86